stable/goldilocks/README.md

```diff
@@ -1,5 +1,5 @@
 <div align="center">
-<a href="https://github.com/FairwindsOps/goldilocks"><img src="logo.svg" height="150" alt="Goldilocks" style="padding-bottom: 20px" /></a>
+<a href="https://github.com/FairwindsOps/goldilocks"><img src="https://raw.githubusercontent.com/FairwindsOps/goldilocks/master/pkg/dashboard/assets/images/goldilocks.svg" height="150" alt="Goldilocks" style="padding-bottom: 20px" /></a>
 <br>
```

Comment:
This image is broken when I view the file, just wanted to mention it in case that was an issue.

Reply:
I think that's just SVG being funky when viewed directly in the browser. The helm chart.yaml spec allows for svg to be used in the `icon` field though, so I think it should be fine.